lib/ddr/contacts.rb

```diff
@@ -0,0 +1,27 @@
+module Ddr
+  module Contacts
+    extend ActiveSupport::Autoload
+
+    autoload :Contact
+
+    @@contacts = {}
+
+    def self.get(contact_slug)
+      @@contacts[contact_slug]
+    end
+
+    def self.contacts
+      @@contacts
```

Comment:
Instead of defining `@@contacts` and `self.contacts`, I would do this:

```ruby
class << self
  attr_accessor :contacts
end

self.contacts = OpenStruct.new
```